src/modules/sub-items/sub.items.module.js

```diff
@@ -293,72 +431,91 @@ export default class SubItemsModule extends Component {
         this.setState(() => ({ activeView }));
     }
 
-    toggleItemSelect(location, isSelected) {
-        const { selectedLocationsIds } = this.state;
-        const updatedSelectedItemsIds = new Set(selectedLocationsIds);
+    /**
+     * Returns data which we want to cache for selected item
+     * Page data might be discarded but selection of the item should stay,
+     * thus we need item data to perform operations such as bulk delete or bulk move
+     *
+     * @param {{content: Object, location: Object}} item
+     */
+    getItemDataToCache(item) {
+        const { contentTypesMap } = this.state;
+        const notAvailableLabel = Translator.trans(/*@Desc("N/A")*/ 'content_type.not_available.label', {}, 'sub_items');
+        const contentType = contentTypesMap[item.content.ContentType._href];
+        const contentTypeName = contentType ? contentType.names.value[0]['#text'] : notAvailableLabel;
```

Comment:
This will return the name in the first language. The translation will not work.

Reply:
Hmm, I took this piece of code from somewhere else, we should use `window.eZ.adminUiConfig.contentTypeNames` now?

Reply:
Yes